lib/process-services-cloud/src/lib/models/data-column-custom-data.ts

```diff
@@ -1,3 +1,21 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const PROCESS_LIST_CUSTOM_VARIABLE_COLUMN = 'process-variable-column';
```

Comment:
Removed `ColumnDataType` enum from `column-data-type.model.ts` and moved value to single `const` (we suppose to have `task-variable-column` but requirements were changed, showing process variables on task list)